bench/report.txt

```diff
@@ -5,7 +5,7 @@ Adding jobs
 Running jobs
 ....................................................................................................
 
-Adding 10000 Jobs Time:   1.6360s
-Running 10000 Jobs Time: 11.1720s
+Adding 10000 Jobs Time:   1.5895s
+Running 10000 Jobs Time: 11.3055s
```

Comment:
@kellyredding - My computer does seem to be going slower today when running jobs, but I think there is about a 0.05 - 0.1s slowdown overall with this change.

I don't know why adding jobs was faster, but there is a variance 0.05s (+ or -) with these benchmarks in general.